clusterloader2/testing/density/legacy/config.yaml

```diff
@@ -17,6 +17,7 @@
 {{$MIN_LATENCY_PODS := 500}}
 {{$MIN_SATURATION_PODS_TIMEOUT := 180}}
 {{$ENABLE_CHAOSMONKEY := DefaultParam .ENABLE_CHAOSMONKEY false}}
+{{$USE_SIMPLE_LATENCY_QUERY := DefaultParam .USE_SIMPLE_LATENCY_QUERY false}}
```

Comment:
legacy/ files shouldn't be use

Instead - we should fix the configs to use appropriate branches from perf-tests repo

@mm4tt @oxddr

Reply:
+1, @oxddr, can we switch 1.13 tests to use 1.13 branch of perf-test? Is there anything left to do before we do that?

Reply:
release-1.13 was reverted to same state as release-1.14, so that should solve the issue with older test

independently, other test are failing due to unknown metric:
https://k8s-testgrid.appspot.com/sig-scalability-node#node-throughput
https://k8s-testgrid.appspot.com/sig-scalability-experiments#storage

I have updated the PR